frontend/pages/admin/IntegrationsPage/cards/Mdm/Mdm.tsx

```diff
@@ -285,7 +285,7 @@ const Mdm = (): JSX.Element => {
       {showEditTeamModal && (
         <EditTeamModal
           onCancel={toggleEditTeamModal}
-          onEdit={toggleEditTeamModal}
+          currentDefaultTeamName={mdmAppleBm?.default_team ?? "No default team"}
```

Comment:
the way this is set up you will never had an undefined value for `currentDefaultTeamName`, yet in `EditTeamModal` we are still handing undefined cases. I'd suggest we either type the prop as only a string, or remove the `"No default team"` prop assignment here.